pages/dkp/kommander/2.2/DKP-insights/architecture-overview/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insights architecture overview
+title: Insights architecture overview
+excerpt: Insights architecture overview
+menuWeight: 10
+beta: false
```

Comment:
There are 2 `beta: false` in this section.

Reply:
The million-dollar question: does this make for a double-negative = a positive, or is it double-plus NOT beta???
Thank you, removed one!